bin/cml-runner.js

```diff
@@ -17,29 +17,31 @@ const {
 
   RUNNER_PATH = `${WORKDIR_BASE}/${NAME}`,
   RUNNER_IDLE_TIMEOUT = 5 * 60,
-  RUNNER_DESTROY_DELAY = 30,
+  RUNNER_DESTROY_DELAY = 10,
   RUNNER_LABELS = 'cml',
   RUNNER_NAME = NAME,
   RUNNER_SINGLE = false,
   RUNNER_REUSE = false,
+  RUNNER_RETRY = false,
```

Comment:
I'd still recommend making this an int in minuted

```suggestion
  RUNNER_RETRY = 72 * 60 - 5,
```